includes/export/abstract-wc-csv-exporter.php

```diff
@@ -296,7 +296,7 @@ protected function export_row( $row_data, $key, $buffer ) {
 	 * @return int
 	 */
 	public function get_limit() {
-		return $this->limit;
+		return apply_filters( "woocommerce_product_export_get_limit", $this->limit, $this->export_type);
```

Comment:
Now should works.
But need some coding style fixes.

- [ ] Need to replace double quotes to single quotes.
- [ ] Missing an empty space after `$this->export_type`.

Reply:
In fact, better change to:

```php
return apply_filters( "woocommerce_{$this->export_type}_export_batch_limit", $this->limit, $this );
```